networks/meld/nfts/0xe1d347431d26e88C4Da85D4dE3f9B0B9326F12a6.json

```diff
@@ -0,0 +1,18 @@
+{
+  "tokenTicker": "mepe",
+  "tokenAddress": "0xe1d347431d26e88C4Da85D4dE3f9B0B9326F12a6",
+  "tokenColorTheme": 1,
+  "tokenType": "ERC721",
+  "collectionName": "mepe theme",
+  "groupedUnder": "",
+  "collectionIcon": "/static/undefined.jpg",
+  "authorsName": "mepe",
+  "authorAddress": "",
+  "maxSupply": "",
```

Comment:
invalid value for  maxSupply

Reply:
Ah sorry , i thought it would be nice to leave it none. Because the nft has unlimited supply